pkg/motor/query.go

```diff
@@ -6,45 +6,10 @@ import (
 
 	mt "github.com/sonr-io/sonr/third_party/types/motor"
 	bt "github.com/sonr-io/sonr/x/bucket/types"
+	st "github.com/sonr-io/sonr/x/schema/types"
 )
 
-func (mtr *motorNodeImpl) QueryBucket(req mt.QueryWhereIsRequest) (*mt.QueryWhereIsResponse, error) {
```

Comment:
Will this break speedway?

Reply:
Not until speedway upgrades version, it will be a simple rename